.github/workflows/tests.yml

```diff
@@ -131,7 +131,13 @@ jobs:
         run: pip install .[all,dev]
 
       - name: Run
-        run: pytest --cov-report xml:coverage.xml --cov="haystack" -m "unit" test/${{ matrix.topic }}
+        run: pytest --cov-report xml:coverage.xml --cov="haystack" -m "unit" --ignore=test/document_stores/test_elasticsearch8.py test/${{ matrix.topic }}
+
+      - name: Run Elasticsearch 8 unit tests
+        if: matrix.topic == 'document_stores'
+        run: |
+          pip install -U .[elasticsearch8]
+          pytest --cov-report xml:coverage.xml --cov="haystack" --cov-append -m "unit" test/document_stores/test_elasticsearch8.py
```

Comment:
This kind of exceptions to tests are dangerous and makes it harder to read the workflow. 
Also they tend to spread and in a short time we'll have more of these.

Let's find a different solution.